src/main/java/io/sentry/autoinstall/SentryInstaller.java

```diff
@@ -3,14 +3,17 @@
 import static io.sentry.autoinstall.Constants.SENTRY_ARTIFACT_ID;
 import static io.sentry.autoinstall.Constants.SENTRY_GROUP_ID;
 
+import java.io.IOException;
 import java.util.List;
+import java.util.Properties;
 import org.apache.maven.model.Dependency;
 import org.eclipse.aether.artifact.Artifact;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
 public class SentryInstaller {
-  public static final String SENTRY_VERSION = "6.28.0";
+
+  private static final String SENTRY_FALLBACK_VERSION = "6.28.0";
```

Comment:
```suggestion
  private static final String SENTRY_FALLBACK_VERSION = "6.31.0";
```
`m`

Reply:
Hmm we'd probably have to bump this version constantly for it to make actual sense because otherwise you'd end up with an outdated SDK at some point which might be harder to debug than simply failing early.

Reply:
Ok, then I'll catch and log an error. This actually shouldn't happen anyway, as the properties file is bundled with the jar.